deploy/base/apps/fluentbit/05-daemonset.yaml

```diff
@@ -31,7 +31,7 @@ spec:
       terminationGracePeriodSeconds: 10
       containers:
         - name: fluentbit
-          image: acraks3i2qzkkxofr7c.azurecr.io/fluent/fluent-bit:1.9.5
+          image: "acraks3i2qzkkxofr7c.azurecr.io/fluent/fluent-bit:1.9.5"
```

Comment:
Since this is base, should it point to a simple dockerhub image instead of an acr specific one?
i.e. `fluent/fluent-bit:1.9.5`